CODEOWNERS

```diff
@@ -25,23 +25,14 @@ CODEOWNERS @MrAlias @MadVikingGod @pellared
 detectors/aws/                                                          @open-telemetry/go-approvers @Aneurysm9
 detectors/gcp/                                                          @open-telemetry/go-approvers @dashpole
 
-exporters/autoexport                                                    @open-telemetry/go-approvers @MikeGoldsmith
+exporters/autoexport                                                    @open-telemetry/go-approvers @pellared
```

Comment:
@MikeGoldsmith I noticed you do not have `Write` permissions and that basically it means that "Codeowners" do not work for you 😬 Are you fine if I take the code ownership as anyway I was doing most of the recent changes in `autoexport`?

Reference https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners